Prox/Prox/Utilities/AnalyticsUtilities.swift

```diff
@@ -17,8 +17,10 @@ class Analytics {
             print("No Flurry key! Not collecting analytics.")
             return
         }
+
+        // Deprecated, but no Swift alternative right now
+        Flurry.setSessionContinueSeconds(30)
         Flurry.startSession(flurryKey)
```

Comment:
I tested this "30 seconds" continuation parameter with the Swift call and verified that it still works on the Flurry live dashboard.